test/Form_test.js

```diff
@@ -415,23 +416,6 @@ describe("Form", () => {
     });
   });
 
-  describe("Default value handling on clear", () => {
```

Comment:
This is something that is technically broken with my change, but it only sort of half worked before anyway. The responsibility should be on the component providing data to make sure it contains default data, if necessary.